File: DDYDLS_CineClubLocalModel/Models/Rating.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DDYDLS_CineClubLocalModel.Models
{
    public class Ratings
    {
        public int Id_Rating { get;set; }
        public int ID_User { get; set; }
        public int Id_Movie { get; set; }
        public int Rating { get; set; }

        public DateTime Date { get; set; }

        public int Approbate { get; set; }

        public string Commentary { get; set; }

        public string Username { get; set; }
    }
}

File: DDYDLS_CineClubFrontEnd/src/app/models/rating.ts
export class Rating {
    id_rating : number;
    id_movie : number;
    id_user : number;
    ratings : number;
    date : Date;

    
    constructor(rating : Rating) {
    this.id_rating = rating.id_rating;
     this.id_movie = rating.id_movie;
     this.id_user = rating.id_user;
     this.ratings = rating.ratings;
     this.date = rating.date;
    }
}
